build_barback/lib/src/transformer/transformer.dart

```diff
@@ -60,12 +60,11 @@ class BuilderTransformer implements Transformer, DeclaringTransformer {
       return;
     }
 
-    // Run the build step.
-    var logger = new Logger('${input.id}');
-    Logger.root.level = Level.ALL;
-    var logSubscription = logger.onRecord.listen((LogRecord log) {
-      if (log.loggerName != logger.fullName) return;
+    hierarchicalLoggingEnabled = true;
 
+    var logger = new Logger.detached('BuilderTransformer-${input.id}');
```

Comment:
Ah nice, never heard of this. Do we really need to enable hierarchical logging then?